docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -178,6 +178,33 @@ How to **fix** it:
 2. **Scale up distributors**<br />
    Scaling up distributors will lower the number of in-flight push requests per distributor.
 
+### MimirIngesterReachingInflightPushRequestLimit
+
+This alert fires when the `cortex_ingester_inflight_push_requests_summary` per ingester instance limit is enabled and the actual number of in-flight push requests is approaching the set limit. Once the limit is reached, push requests to the ingester will fail (5xx) or 429 for new requests, while existing in-flight push requests will continue to succeed.
```

Comment:
> Once the limit is reached, push requests to the ingester will fail (5xx) or 429 for new requests, while existing in-flight push requests will continue to succeed.

This is not true. Ingester does not reject requests based on "summary" metric, but based on real number of inflight requests at the moment. That number fluctuates a lot.

Reply:
should we update ingester to reject requests base on summary instead?